pkg/tasks/prerequisites.go

```diff
@@ -0,0 +1,37 @@
+package tasks
+
+import (
+	alertsv1alpha1 "github.com/monstorak/monstorak/pkg/apis/alerts/v1alpha1"
+	"github.com/monstorak/monstorak/pkg/common"
+	"github.com/monstorak/monstorak/pkg/prometheus"
+)
+
+// Prerequisites needed for Monitoring to work with Storage
+func Prerequisites(instance *alertsv1alpha1.StorageAlert) error {
+	namespace := instance.Spec.StorageAlert.PrometheusNamespace
+	// Check if Namespace exists
+	_, err := common.GetNamespace(namespace)
+	if err != nil {
+		return err
+	}
+	// Check if Namespace has required labels
+	label := make(map[string]string)
+	label["openshift.io/cluster-monitoring"] = "true"
```

Comment:
Do we need to create this label if doesn't exist?

Reply:
Yes. Since, it is a prerequisite we will just check if it is met.

Reply:
Ack